sdk/lib/rtl/critical.c

```diff
@@ -28,6 +28,10 @@ extern HANDLE LdrpShutdownThreadId;
 
 /* FUNCTIONS *****************************************************************/
 
+#define CRITSECT_HAS_DEBUG_INFO(CriticalSection) \
+    (((CriticalSection)->DebugInfo != NULL) && \
```

Comment:
Isn't this also the field those wine-hacked critical sections have the filename stored in?

Reply:
No, they store it in a field in the debug info: https://git.reactos.org/?p=reactos.git;a=blob;f=sdk/include/psdk/winbase.h;hb=56d95154eeaf2e96d5f3c6cd47baa03e50435235#l879